packages/in_app_purchase/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 ## 0.3.0
 
 * Migrate the `Google Play Library` to 2.0.3.
+     * **[Breaking Change]:**  Added `enablePendingPurchases` in `InAppPurchaseConnection`. The application has
```

Comment:
nit: I think a link to the Play documentation on `enablePendingPurchases` would be useful here.